aptos-move/framework/aptos-stdlib/sources/cryptography/bn254_algebra.move

```diff
@@ -0,0 +1,915 @@
+/// This module defines marker types, constants and test cases for working with BN254 curves using the generic API defined in `algebra.move`.
+/// BN254 was sampled as part of the [\[BCTV14\]](https://eprint.iacr.org/2013/879.pdf) paper .
+/// The name denotes that it is a Barreto--Naehrig curve of embedding degree 12,
+/// defined over a 254-bit (prime) field. The scalar field is highly 2-adic.
```

Comment:
Can you qualify the "highly 2-adic" by saying it supports 2^28 roots of unity?

i.e., since, AFAIK, `(21888242871839275222246405745257275088548364400416034343698204186575808495617-1) mod 2^28 = 0`

Reply:
Thank you. resolved then.